@AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/fn_getInput.sqf

```diff
@@ -17,10 +17,31 @@ Author:
 	BradMick
 ---------------------------------------------------------------------------- */
 params ["_heli"];
+#include "\fza_ah64_systems\headers\systems.hpp"
 
-private _collectiveVal = _heli animationSourcePhase "collective";
+private _config            = configFile >> "CfgVehicles" >> typeof _heli >> "Fza_SfmPlus";
+private _pitchTorque       = getNumber (_config >> "cyclicPitchTorque");
+private _rollTorque        = getNumber (_config >> "cyclicRollTorque");
+private _yawTorque         = getNumber (_config >> "pedalYawTorque");
 
-private _collectiveOut = 0.0;
+private _tailRtrFixed      = false;
+
+private _priHydPumpDamage  = _heli getHitPointDamage "hit_hyd_pripump";
+private _priHydPSI         = _heli getVariable "fza_systems_priHydPsi";
+
+private _utilHydPumpDamage = _heli getHitPointDamage "hit_hyd_utilpump";
+private _utilLevel_pct     = _heli getVariable "fza_systems_utilLevel_pct";
+
+private _accState 		   = _heli getVariable "fza_systems_accState";
+
+private _collectiveVal     = _heli animationSourcePhase "collective";
+private _cyclicFwdAft      = _heli animationSourcePhase "cyclicForward";
+private _cyclicLeftRight   = _heli animationSourcePhase "cyclicAside";
+private _pedalLeftRight    = (inputAction "HeliRudderRight") - (inputAction "HeliRudderLeft");
```

Comment:
Could we use a rudder animation source? I am guessing because we don't use it properly rn we can't but could we fix it?

Reply:
There is no animation source for helicopter rudder/pedals in ARMA except for AFM.

Reply:
Noted, thank you